### PR TITLE
Optimize UI around session recordings

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Small fixes for session recordings UI (for instance links in `Tooltip`s don't work, because tooltip content can't be clicked, the hoverable element must be a link instead).

| Before | After |
| --- | --- |
| Screen Shot 2021-08-28 at 01 55 18 | Screen Shot 2021-08-28 at 01 49 49 |
| Screen Shot 2021-08-28 at 01 36 24 | Screen Shot 2021-08-28 at 01 36 46 |

### CODE REVIEW

File: frontend/src/scenes/sessions/SessionsView.tsx [hunk 1]
```diff
@@ -159,22 +147,22 @@ export function SessionsView({ personIds, isPersonPage = false }: SessionsTableP
...
-                        Recordings
                         {currentTeam?.session_recording_opt_in ? (
-                            <QuestionCircleOutlined className="info-indicator" />
+                            <>
+                                Recordings
+                                <QuestionCircleOutlined className="info-indicator" />
+                            </>
                         ) : (
-                            <PoweroffOutlined className="info-indicator text-warning" />
+                            <Link to={urls.projectSettings()}>
```
Comment (mariusandra) [inline frontend/src/scenes/sessions/SessionsView.tsx:162]: Would be nice to link directly to the tab. Not sure if it should be:
```jsx
  <Link to={`${urls.projectSettings()}#session-recording`}>
```

... or `urls.projectSettings({ anchor: 'session-recording' })` 

... or any other variation.

**Review by mariusandra — CHANGES_REQUESTED** [review]
Changes make sense, yet found some issues:

1. The button gets cut off in the table:
image
image

Especially visible with smaller resolutions:

image

2. The `ms` probably isn't useful to this precision:

image

3. (inline comment) that we should scroll to the session recordings part of the project page when clicking the link.

**Comment by Twixes** [comment]
Sure @mariusandra

**Comment by Twixes** [comment]
Sure @mariusandra

File: frontend/src/lib/utils.tsx [hunk 2]
```diff
@@ -517,6 +517,7 @@ export function colonDelimitedDuration(d: string | number | null | undefined, nu
         m = Math.floor(s / 60)
         s -= m * 60
     }
+    s = Math.round(s)
```
Comment (paolodamico) [inline frontend/src/lib/utils.tsx:520]: Maybe worth adding a test that properly captures this?

**Review by paolodamico — APPROVED** [review]
lgtm!

**Pull request merged**